qcodes/instrument_drivers/agilent/E8267C.py

```diff
@@ -84,9 +84,9 @@ def __init__(self, name: str, address: str, **kwargs: Any) -> None:
 
     # functions to convert between rad and deg
     @staticmethod
-    def deg_to_rad(angle_deg: numbertypes) -> np.float:
+    def deg_to_rad(angle_deg: numbertypes) -> "np.floating[Any]":
```

Comment:
Consider replacing with something not Any https://numpy.org/devdocs/reference/typing.html#number-precision